### ISSUE TITLE
Illegal instruction: 4 / Segmentation fault on iOS 5.x

### ISSUE THREAD

**Issue by MungoBerry** [angelXwind/AppSync#36]
Hi,

I'm trying to install AppSync Unified via Cydia but a segmentation fault occurs:

> subprocess installed post-installation script was killed by signal (Segmentation fault
> Sub-process /usr/libexec/cydia/cydo returned on error code (1)

Any hint ?
Thank you!

**Comment by angelXwind** [comment]
You likely installed AppSync Unified from a third-party repository, as I can confirm that this does not occur on my iOS 5.1.1 testing device (iPod touch 3G, N18AP).

Make sure you are installing the package `net.angelxwind.appsyncunified` from my repo, https://cydia.angelxwind.net/.

**Comment by angelXwind** [comment]
This bug has been fixed as of version 31.0. Please update to 31.0.

If you need help clearing the half-broken install on iOS 5, run these commands in an SSH session as `root`:

```
rm -fv /var/lib/dpkg/info/net.angelxwind.appsyncunified.postinst
rm -fv net.angelxwind.appsyncunified.prerm
dpkg --purge --force-all net.angelxwind.appsyncunified
```

If you're curious, this issue was actually caused by an undocumented change in Theos caused by the Makefile cleanup that occurred… some time ago — manually specifying iOS deployment version fixes the issue, though I'm going to try to fix the underlying issue in Theos later when I have the time.

**Comment by MungoBerry** [comment]
Now works!
Thank you for your time.
